parse_test.go

```diff
@@ -0,0 +1,192 @@
+package github
+
+import (
+	"flag"
+	"os"
+	"strconv"
+	"testing"
+)
+
+var (
+	tmpKeyFile string = "/tmp/key.pem"
+)
+
+func prepare() {
+	instId = 12345
```

Comment:
```suggestion
func prepare() {
	// Remove the artifacts from the previous run
	cleanup()
	instId = 12345
```
After looking over the tests -- I'm wondering if, it would make sense to do the "cleanup()" as part of the preparation.

It would guarantee that your tests are in a predictable state _before_ they run.

Of course, using `defer cleanup()` - in theory could do the same, but if a unit test "forgets" to add the cleanup() - it would break other (unrelated) tests.  This would be confusing to debug.

Reply:
Plus, you could then remove the `defer cleanup() ` from all the tests.